project/Deps.scala

```diff
@@ -55,6 +63,18 @@ object Deps {
     val postgres = "org.postgresql" % "postgresql" % V.postgresV
     val uJson = "com.lihaoyi" %% "ujson" % V.uJsonV
 
+    // serializing to and from JSON
+    val uPickle = "com.lihaoyi" %% "upickle" % V.uPickleV
+
+    // make akka-http play nice with upickle
+    val akkaHttpUpickle = "de.heikoseeberger" %% "akka-http-upickle" % V.akkaHttpUpickleV
+
+    // parsing of CLI opts and args
+    val scopt = "com.github.scopt" %% "scopt" % V.scoptV
+
+    // HTTP client lib
+    val sttp = "com.softwaremill.sttp" %% "core" % V.sttpV
```

Comment:
do we really need _another_ http client lib?

Reply:
Valid point, but in that case I'd say its a better idea to scrap akka-http as a client lib. 

Classpath of `akka-http` + `akka-stream` (need both):
```bash
$ coursier resolve com.typesafe.akka:akka-stream_2.12:2.5.23
com.typesafe:config:1.3.3:default
com.typesafe:ssl-config-core_2.12:0.3.7:default
com.typesafe.akka:akka-actor_2.12:2.5.23:default
com.typesafe.akka:akka-protobuf_2.12:2.5.23:default
com.typesafe.akka:akka-stream_2.12:2.5.23:default
org.reactivestreams:reactive-streams:1.0.2:default
org.scala-lang:scala-library:2.12.8:default
org.scala-lang.modules:scala-java8-compat_2.12:0.8.0:default
org.scala-lang.modules:scala-parser-combinators_2.12:1.1.1:default

$ coursier resolve com.typesafe.akka:akka-http_2.12:10.1.8
com.typesafe.akka:akka-http-core_2.12:10.1.8:default
com.typesafe.akka:akka-http_2.12:10.1.8:default
com.typesafe.akka:akka-parsing_2.12:10.1.8:default
org.scala-lang:scala-library:2.12.8:default
```

classpath of Sttp: 
```bash
$ coursier resolve com.softwaremill.sttp:core_2.12:1.1.13
com.softwaremill.sttp:core_2.12:1.1.13:default
org.scala-lang:scala-library:2.12.5:default
```